### PR TITLE
feat: add kubectl to devcontainer

### PR DESCRIPTION
**Pull request by patheard**
# Summary
Add kubectl, [kubectx](https://github.com/ahmetb/kubectx/) and command auto-complete to
the devcontainer.  This lets us monitor how the cluster
is behaving during API performance testing.

Also adds the [GitHub Copilot](https://copilot.github.com/) VS Code extension.

### CODE REVIEW

File: .devcontainer/scripts/notify-dev-entrypoint.sh [hunk 1]
```diff
@@ -16,6 +16,12 @@ echo -e "alias l='exa -alh'" >> ~/.bashrc
 echo -e "alias ll='exa -alh@ --git'" >> ~/.bashrc
 echo -e "alias lt='exa -al -T -L 2'" >> ~/.bashrc
 
+# Kubectl aliases and command autocomplete
+echo -e "alias k='kubectl'" >> ~/.bashrc
+echo -e "source <(kubectl completion bash)" >> ~/.bashrc
+echo -e "complete -F __start_kubectl k" >> ~/.bashrc
+echo -e "source /usr/share/bash-completion/bash_completion" >> ~/.bashrc
```
Comment (jimleroyer) [inline .devcontainer/scripts/notify-dev-entrypoint.sh:23]: Could be nice to use oh my zsh instead, dump starship. The former offers so much for auto completion and plugins.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -33,7 +33,8 @@
 		"visualstudioexptteam.vscodeintellicode",
 		"yzhang.markdown-all-in-one",
 		"ms-ossdata.vscode-postgresql",
-		"googlecloudtools.cloudcode"
+		"googlecloudtools.cloudcode",
+		"GitHub.copilot"
```
[the remark this answers is not in the document]
Comment (patheard) [reply .devcontainer/devcontainer.json:37]: Correct - if it proves to be annoying for people without accounts, I more than happy to remove and only install myself.

File: .devcontainer/scripts/notify-dev-entrypoint.sh (hunk 1 above)
Comment (patheard) [reply .devcontainer/scripts/notify-dev-entrypoint.sh:23]: I'm definitely game for switching to `zsh`, I had just thought there was a `starship` preference.  I'll get that added!

Comment (jimleroyer) [reply .devcontainer/scripts/notify-dev-entrypoint.sh:23]: I like starship as it's cross platform and written in Rust (I contributed a bit to it) -- but oh-my-zsh might play much better for devcontainers and all of its plugins. 🙃

**Pull request merged**